lighthouse-core/fraggle-rock/gather/runtime-controller.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const pageFunctions = require('../../lib/page-functions.js');
+
+class RuntimeController {
```

Comment:
> Sure, ideas?
>
> ExecutionContextController
> EvaluateAsyncController
> were also under consideration

those work for me, though I'm generally in the "Drop the 'Controller'; it's cleaner" camp 

(alternatively: "if everything's a `*Controller`, then...there's no reason to keep that part of the name" :)